src/slices/PreviousEdition.tsx

```diff
@@ -0,0 +1,47 @@
+import DefaultButton from "@/components/DefaultButton";
+import { useTheme, Box, Center, Flex, VStack, Heading, Link, AspectRatio } from "@chakra-ui/react";
+
+
+const PreviousEdition = () => {
+    const theme = useTheme();
+
+    const content = {
+        text: 'Zobacz poprzednie prelekcje',
+        buttonText: 'PRZEJDŹ DO NAGRAŃ',
+        youtubeUrl: 'https://www.youtube.com/embed/jMk-6SDVXC8',
+        previousEditionUrl: 'https://16.sesja.linuksowa.pl/'
+    }
+
+
+    return <Box
+        display="flex"
+        width="100%"
+        bg={theme.colors.primary}
+        minHeight="70vh"
+        opacity="90%"
+        alignItems="center"
+        justifyContent="center">
+
+        <Flex width="95%"
+            marginY="auto"
+            direction={{ base: "column", lg: "row" }}
+            >
```

Comment:
Zwykle jak się pisze w Reactcie to `>` wyrównujesz do zagłębienia początku taga.

Reply:
`npm run format` powinien to poprawić

Reply:
Poprawione